pynest/examples/sudoku/plot_progress.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+#
+# plot_progress.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+r"""Generate a GIF of the network solving a Sudoku puzzle 
+----------------------------------------------------------------
+This scripts takes one of the .pkl files generated by 
+:doc:`sudoku_solver.py` and generates a GIF showing the progress
+of the network solving the puzzle. 
+
+Note that the script generates the images individually, storing
+them to disk first, assembling them into a GIF and then,
+by default, deleting the images and folder.
+
+For creating the individual images, PIL is used. Because the GIF creation 
+tool in PIL is faulty, imageio is required for that step.
+
+See Also
+~~~~~~~~
+
+:doc:`sudoku_solver`
+:doc:`helpers`
```

Comment:
```suggestion

:doc:`helpers`
```

Reply:
Thank you, I am very much open to that. Let me know if you need my help.